sdk/cosmos/microsoft-azure-cosmos/src/main/java/com/azure/data/cosmos/internal/changefeed/ServiceItemLease.java

```diff
@@ -62,7 +62,7 @@ public ServiceItemLease withId(String id) {
         return this;
     }
 
-    @JsonIgnore
+    @JsonProperty("_etag")
```

Comment:
when a class is annotated with jackson annotation, jackson uses reflection for serialization/deserialization to json.

reflection is expensive and that being said, If there are pre existing classes in the code which needs to be serialized/deserialized to json we should ideally use a jackson serializer rather than annotating the class with jackson annotation.

(not necessarily in the scope of this PR, but something to be addressed outside of this PR).

Reply:
In this case the "_etag" field was not passed through the serialization and that resulted in "IF MATCH" condition to fail when updating the lease document.